### DISCUSSION TITLE
JC2432W38 Variant

### DISCUSSION THREAD

**Discussion by gpena208777** [witnessmenow/ESP32-Cheap-Yellow-Display#244] [General]
Hi, I have the USB C Capacitive version that has JC2432W328(Type in Title still exists) printed on the rear. I cannot get any screen display with any example or even the Web Flasher. It's still a CYD. Could this be why im having so many issues?

**Comment by TheNitek** [comment]
This is not a CYD, so the examples here won't work out of the box (you probably have to adjust at least the PIN defintions)

**Comment by Cryptorial0** [comment]
did you ever get it to work?
